plugin/mutate/data/source.js

```diff
@@ -583,22 +596,32 @@ function init_legend() {
    document.getElementById("legend4_key").innerHTML = key_traverse_muts_up;
    document.getElementById("legend5_key").innerHTML = key_toggle_show_mutant;
 }
-function init_filter_kind() {
-    var table = document.getElementById("filter_kind");
 
-    for (var i=0; i < g_mut_kindGroup_map.length; i++) {
-        var row = table.insertRow(i);
-        row.innerHTML = `<td><input id="`+g_mut_kindGroup_map[i]+ `" type="checkbox" onclick='click_filter_kind(`+i+`)' checked />
-                        <span class="xx_label">`+g_mut_kindGroup_map[i]+`</span></td>`;
+function hide_testcases(){
+  var num = document.getElementById("num_testcases");
+
+  if (num.style.display === "none") {
+      num.style.display = "block";
+    } else {
+      num.style.display = "none";
     }
 }
-function init_filter_status() {
-    var table = document.getElementById("filter_status");
 
-    for (var i=0; i<g_mut_st_map.length; i++) {
-        var row = table.insertRow(i);
-        row.innerHTML = `<td><input id="`+g_mut_st_map[i]+ `" type="checkbox" onclick='click_filter_status(`+i+`)' checked />
-        <span class="xx_label">`+g_mut_st_map[i]+`</span></td>`;
-    }
-    return;
+function hide_kind(){
```

Comment:
leftover code from experimenting?

Reply:
We agreed on removing filter by kind and filter by status from the filereport tool. Should I remove all past functions that are were used for them but are not used anymore?